packages/vue-vocabulary/package.json

```
@@ -12,6 +12,7 @@
     "storybook": "vue-cli-service storybook:serve --ci -s ./src/assets -p 8082",
     "build:documentation": "npm run build && vue-cli-service storybook:build -s ./src/assets -o ./docs"
   },
+  "main": "vue-vocabulary.umd.min.js",
```

```suggestion
  "main": "vue-vocabulary.common.js",
  "unpkg": "vue-vocabulary.umd.min.js",  
```

I would recommend we default to the `cjs` modules, but specify `unpkg` uses the `umd` build.

We can assume most non-cdn consumers of vocabulary are using the library in an app where their code is being compiled by a a build step.

---

I added the requested changes.

I wonder what is the reason for using `cjs` modules? At first, I thought it was probably the size, but `umd`, and especially `umd.min`, are not larger. Is it more performant?

---

Let's chat about modules on our next call. Could be fun. There's a lot going on there—a bit too much for a message here. I will say, there's nothing _wrong_ about using `umd` as the `main`...right now, it's mostly just a _convention_ to use `cjs` in the main field.